spec/dynamoid/transaction_write/update_spec.rb

```diff
@@ -224,6 +224,134 @@
           end
         }.to output('saving updating updated saved ').to_stdout
       end
+
+      context 'adds' do
+        it 'to nil' do
```

Comment:
The test title doesn't describe the behaviour - I suppose if current value is `nil` then `0` is used.

Reply:
Clarified with 'to nil which defaults to zero'.